aldryn_search/search_indexes.py

```diff
@@ -49,8 +49,14 @@ def get_search_data(self, obj, language, request):
         text_bits = []
 
         for base_plugin in plugins:
-            plugin_text_content = self.get_plugin_search_text(base_plugin, request)
-            text_bits.append(plugin_text_content)
+            try:
```

Comment:
I don't think this is the way to go, please handle this on the project level. If indexing fails here is because of an issue in the application that's being indexed which needs dev intervention.